README.md

```diff
@@ -0,0 +1,286 @@
+| **WANT\_RESPONSE\_SIGNED**                  | Set this to `False` if you don't want your provider to sign the response.                                                                                                                                                                                                                                                                                                                                                                                   | `bool`           | `True`                                                                                                                                   |                                                          |
+| **ACCEPTED\_TIME\_DIFF**                    | Sets the [accepted time diff](https://pysaml2.readthedocs.io/en/latest/howto/config.html#accepted-time-diff) in seconds                                                                                                                                                                                                                                                                                                                                     | `int` or `None`  | `None`                                                                                                                                   |                                                          |
+| **ALLOWED\_REDIRECT\_HOSTS**                | Allowed hosts to redirect to using the `?next=` parameter                                                                                                                                                                                                                                                                                                                                                                                                   | `list`           | `[]`                                                                                                                                     | `['https://app.example.com', 'https://api.exmaple.com']` |
+
+
+## PKI for JWT
+
+If you want to use your PKI key-pair to sign JWT tokens, you might use these fields instead `JWT_SECRET`:
+
+```python
+{
+    ...
+    'USE_JWT': True,
+    'JWT_ALGORITHM': 'RSA256',
+    'JWT_PRIVATE_KEY': '--- YOUR PRIVATE KEY ---',
```

Comment:
Is it worth mentioning the type of supported keys? i.e. RSA, ECSDA etc.

Most people use `RSA` but it might be worth clarifying if others are supported or not.